pkgs/development/compilers/rust/binary.nix

```diff
@@ -30,6 +30,7 @@ rec {
       description = "A safe, concurrent, practical language";
       maintainers = with maintainers; [ qknight ];
       license = [ licenses.mit licenses.asl20 ];
+      platforms = platforms.all;
```

Comment:
Why?

Reply:
fails to eval without it
```console
error: attribute 'platforms' missing

       at /home/figsoda/nixpkgs/pkgs/build-support/rust/build-rust-package/default.nix:161:17:

          160|     # default to Rust's platforms
          161|     platforms = rustc.meta.platforms;
             |                 ^
          162|   } // meta;
```

Reply:
Why the bootstrap binaries now require that meta.platforms is set?

Reply:
I had to change how `cargo-auditable` is bootstrapped so `cargo` doesn't depend on itself, though I'm not sure why the change would require boostrap binaries to need `meta.platforms`

Reply:
Is `cargo-auditable` being built with rust/cargo from binary bootstrap instead of the rust/cargo that we've built from source?

Reply:
yes, alternatively we can build cargo with a cargo-auditable built with the binary bootstrap, which would result in one extra cargo-auditable, or we can bootstrap cargo one extra time

Reply:
Not sure which of those two is better really, any reasons to pick one over the other?

If there isn't I guess I'd lean toward bootstrapping cargo an extra time?

Reply:
I figured out an easier way to do this